### DISCUSSION TITLE
А можно сделать так, чтобы правила для iptables/ip6tables добавлялись только когда локальный интерфейс активен?

### DISCUSSION THREAD

**Discussion by Seamas73** [Anonym-tsk/tpws-keenetic#19] [Q&A]
Я прописал в tpws.conf два локальных интерфейса: br0 (Домашняя сеть) и br1 (Гостевая сеть), чтобы гости тоже могли смотреть ютуб из моей локалки. Но, пока гостей нет, я "Гостевую сеть" отключил. При рестарте сервиса после добавления локального интерфейса, добавилось два комплекта правил для iptables/ip6tables.
А можно сделать так, чтобы правила добавлялись только при активации гостевого интерфейса и удалялись после его отключения?
Наверное, придётся писать отдельный скрипт, чтобы регулярно проверять статус гостевого интерфейса и добавлять/удалять правила для iptables/ip6tables при его включении/выключении?

**Comment by Anonym-tsk** [comment]
Технически - можно всё.
Планов таких пока нет.
Заведите пожалуйста Issue - Feature Request, может когда-нибудь сделаем
